idp.go

```diff
@@ -109,6 +110,26 @@ func (idp *IdentityProvider) MetaDataResponse() (string, *Reject) {
 			})
 		}
 	}
+	
+	if idp.Organization != nil {
+		metadata.Organization = idp.Organization
+	}
+
+	if len(idp.ContactPerson) > 0  {
```

Comment:
@sccalabr idp.ContactPerson and idp.Organization  are unresolved references, IdentityProvider struct doesn't have ContactPerson and Organization  as its fields